exercises/simple-cipher/canonical-data.json

```diff
@@ -105,13 +105,22 @@
           "expected": "zzzzzzzzzz"
         },
         {
-          "description": "Can handle messages longer than the key",
+          "description": "Can encode messages longer than the key",
           "property": "encode",
           "input": {
             "key": "abc",
             "plaintext": "iamapandabear"
           },
           "expected": "iboaqcnecbfcr"
+        },
+        {
+          "description": "Can decode messages longer than the key",
+          "property": "decode",
+          "input": {
+            "key": "abc",
+            "plaintext": "iboaqcnecbfcr"
```

Comment:
I highly recommend that this has to be ciphertext, like all the other `decode` cases in this file.

Reply:
Huh, yeah, that slipped by.  I agree should be ciphertext.  I think this would also constitute a major version change as well.

Reply:
Apologies. Missed that one too (the evils of copy+paste!). I'll submit a PR changing to `ciphertext`. I'll bump the version to major also.